server/app.py

```diff
@@ -194,6 +195,16 @@ def player_count():
                                    count=len(APP.waiting_clients))
     return jsonify(response.to_dict())
 
+@APP.route('/api/game_state', methods=['GET'])
+def game_state():
+    client_state_request = ClientGameStateRequest.from_dict(dict(request.args))
+    logging.info('Parsed request: %s', client_state_request)
+
+    client = APP.get_client(client_state_request.client_id)
+    game_id = client.game_id
```

Comment:
maybe we can add an if/else statement here to say if not client.game_id, just return empty or something

Reply:
That and i also forgot the jsonify on the return value

Reply:
Ok I'll wait for your fix